docs/core/compatibility/core-libraries/7.0/cpluspluscli-compiler-version.md

```diff
@@ -39,7 +39,7 @@ This change can affect [source compatibility](../../categories.md#source-compati
 
 ## Recommended action
 
-To continue using C++/CLI with .NET 7, upgrade to Visual Studio 2022 version 17.2 or a later version.
+To continue using C++/CLI with .NET 7, upgrade to Visual Studio 2022 version 17.2 or a later version. Also make sure that the [platform toolset](/cpp/build/how-to-modify-the-target-framework-and-platform-toolset#platform-toolset) is set to v143 or later.
```

Comment:
C++/CLI for .NET 8 is VS 2022 v17.8 or later and toolset v143 or later.

v143 is not required for .NET 7

Reply:
Based on [this comment](https://github.com/dotnet/docs/issues/28542#issuecomment-1505460772), I thought it was required even for .NET 7.